src/browser/provider/built-in/chrome/cdp.js

```diff
@@ -45,18 +44,8 @@ async function setEmulation (runtimeInfo) {
     await resizeWindow({ width: config.width, height: config.height }, runtimeInfo);
 }
 
-async function getScreenshotData (client) {
-    const { visualViewport } = await client.Page.getLayoutMetrics();
-
-    const clipRegion = {
-        x:      visualViewport.pageX,
-        y:      visualViewport.pageY,
-        width:  visualViewport.clientWidth,
-        height: visualViewport.clientHeight,
-        scale:  visualViewport.scale
-    };
-
-    return await client.Page.captureScreenshot({ fromSurface: true, clip: clipRegion });
+export async function getScreenshotData (client) {
+    return await client.Page.captureScreenshot({ format: 'png', fromSurface: true });
```

Comment:
According to https://chromedevtools.github.io/devtools-protocol/tot/Page, `png` is the default value for the `format` parameter and `true` is the default value for `fromSurface` parameter.
It means you can omit them.